picard/ui/options/general.py

```diff
@@ -41,6 +41,7 @@ class GeneralOptionsPage(OptionsPage):
         config.TextOption("persist", "oauth_refresh_token", ""),
         config.BoolOption("setting", "analyze_new_files", False),
         config.BoolOption("setting", "ignore_file_mbids", False),
+        config.BoolOption("setting", "cluster_new_files", False),
```

Comment:
We have a choice to make here. I think the goal here is to help new comers, so setting this to True would make sense, though it would change behavior for "old" users, which is a bad idea.
What do you think?